scripts/docker-run.sh

```diff
@@ -53,6 +55,16 @@ run() {
 
   local -a args=(--rm)
 
+  local cruntime=docker
+  local sudo_cmd
+  if [[ -z "$CI" ]] && command -v podman >/dev/null 2>&1
+  then
+    cruntime=podman
+    # rootless containers are a PITA
+    # https://www.tutorialworks.com/podman-rootless-volumes/
```

Comment:
Is this really needed? Do we test both runtimes? I think it adds complexity while adding no real benefit.

1. Why not use [docker w/ rootless](https://docs.docker.com/engine/security/rootless/
)?
1. Do we want to support singularity containers, too?

Reply:
What's your point? "Please remove this feature"?

The cli of both docker and podman are almost 100% the same. Besides using a different command name there is pretty much no added complexity here.

I want to support both, cause I don't want contributors to go through the hassle of installing podman or docker even though their distro ships the other.

Reply:
Your comment on line 63 made me ask the question

> # rootless containers are a PITA

Additionally, Podman presently only supports running containers on Linux. However, we are building a remote client which can run on Windows and macOS and manage Podman containers on a Linux system via the REST API using SSH tunneling.

Feels like a solution to a problem no one has encountered (e.g., pre-optimization).

Reply:
Well, well, well. This all started cause I didn't want to:

1. install docker 
2. Argue with people about container runtimes

Sigh.